### PR TITLE
Add gs.FLYERS

### PR DESCRIPTION
**Pull request by danielballan**
Closes https://github.com/NSLS-II/wishlist/issues/79

### CODE REVIEW

**Comment by danielballan** [comment]
attn @stuwilkins

File: bluesky/examples.py [hunk 1]
```diff
@@ -115,6 +115,9 @@ def settle(self):
     def stop(self):
         pass
 
+    def trigger(self):
```
Comment (tacaswell) [inline bluesky/examples.py]: I don't think moveables actually have a trigger

File: bluesky/tests/test_simple_api.py [hunk 2]
```diff
@@ -59,3 +59,6 @@ def test_basic_usage():
     tscan(1, 2, 2, time=0.1)
     dtscan(1, 2, 2, time=0.1)
     th2th(1, 2, 2, time=0.1)
+    flyer = FlyMagic('wheee', motor, det1, det2)
```
Comment (tacaswell) [inline bluesky/tests/test_simple_api.py]: I like the name.

File: bluesky/examples.py (hunk 1 above)
Comment (danielballan) [inline bluesky/examples.py]: As I remember it, we allow for the possibility that users might want to set a motor and then trigger that setting as a separate step. Imagine the case where set performs some moderately expensive computation.

Comment (danielballan) [inline bluesky/examples.py]: Also, Movable's interface is a subset of the Readable interface.

Comment (tacaswell) [inline bluesky/examples.py]: oh, fair point.

**Pull request merged**